submissions/the-darkesttt/frogger-task/app.js

```diff
@@ -0,0 +1,323 @@
+const playerSkins = [
+    {
+        id: 'boy',
+        src: 'images/char-boy.png',
+    },
+    {
+        id: 'cat-girl',
+        src: 'images/char-cat-girl.png',
+    },
+    {
+        id: 'horn-girl',
+        src: 'images/char-horn-girl.png',
+    },
+    {
+        id: 'pink-girl',
+        src: 'images/char-pink-girl.png',
+    },
+    {
+        id: 'princess-girl',
+        src: 'images/char-princess-girl.png',
+    },
+];
+
+class Menu {
+    bg;
+    menu;
+    menuTitle;
+    menuDescriprion;
+
+    menuList;
+    menuItem;
+
+    menuScore;
+    menuClose;
+
+    constructor() {
+        this.bg = document.createElement('div');
+        this.bg.classList.add('bg');
+        this.menu = document.createElement('div');
+        this.menu.classList.add('menu');
+        this.menuTitle = document.createElement('h1');
+        this.menuTitle.innerText = 'Game Menu';
+        this.menuDescriprion = document.createElement('p');
+    }
+
+    start() {
+
+        this.menuDescriprion.innerText = 'Choose your character!';
+        this.menuList = document.createElement('ul');
+
+        playerSkins.forEach((item) => {
```

Comment:
You can give more distinctive names to arguments and variables. Like, for example, here you can swap `item` with `skin`. `skin.src` and `skin.id` would give more sense this way, right?